scalars/contributed/chillicream/date.md

```diff
@@ -0,0 +1,128 @@
+# Date — GraphQL Custom Scalar
+
+Author – ChilliCream
+
+Date – 2025-12-24
+
+**License and Copyright**
+
+Copyright © GraphQL contributors. This specification is licensed under
+[OWFa 1.0](https://www.openwebfoundation.org/the-agreements/the-owf-1-0-agreements-granted-claims/owfa-1-0).
+
+# Overview
+
+The `Date` scalar type represents a date in UTC. Unlike `LocalDate`, which
+represents a calendar date without time zone context, `Date` is time-zone-aware
+and always represents the UTC date.
```

Comment:
I'm curious what are the use cases when you'd want to use `Date` that are not covered by either `DateTime` or `LocalDate`?

Reply:
> you'd use `Date` for something like a birth date

I would use `LocalDate` for a birth date. The country you're born in is the context. If I share my date of birth, `1981-09-27` without more context, I'd assume most of my contacts to assume the French timezone. If I want to carry the exact time I was born for legal reasons, I would use something like `1981-09-27T00:14:00+02:00` 

> not only to avoid unnecessary storage, but also to indicate that only the date is meaningful – f.e. there is a time associated with a birth, but it may not be available or relevant.

Then why not use `LocalDate`? My date of birth is `1981-09-27`. Feels weird to send it as `1981-09-26` because some intermediary wants to align everything to UTC.